src/@types/rateLimit.d.ts

```diff
@@ -23,10 +23,10 @@ export interface RedisBucket {
     tokens: number;
     timestamp: number;
 }
-
 export interface RedisWindow {
     currentTokens: number;
-    previousTokens: number;
+    // null if limiter is currently on the initial fixed window
+    previousTokens?: number | null;
```

Comment:
It doesn't look like you need a previous tokens based on the implementation.

Reply:
As we discussed before, this is optional for fixed window, but a must for sliding window.

Reply:
@donjewey can correct me if I'm wrong but I believe his implementation of SlidingWindowCounter was refactored to account for `previousTokens` being required and therefore not needing the null check.

Reply:
That is correct, didn't notice this at all. I was having a hard time with the local and remote commit merge... curious if this is the reason. Thanks for pointing out this minor error. Has it changed already.